pkg/client/unversioned/request_test.go

```diff
@@ -959,9 +962,6 @@ func TestDoRequestNewWayFile(t *testing.T) {
 	requestURL := testapi.Default.ResourcePathWithPrefix("foo/bar/baz", "", "", "")
 	requestURL += "?timeout=1s"
 	fakeHandler.ValidateRequest(t, requestURL, "POST", &tmpStr)
-	if fakeHandler.RequestReceived.Header["Authorization"] == nil {
```

Comment:
The answer to that is, I missed a lot on my search. I will make sure I get them all this time.